### ISSUE TITLE
Police Oversight form: Missing alt text on review page map

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#1500]
![Screen Shot 2019-03-15 at 4.32.48 PM.png](https://images.zenhubusercontent.com/5a5f8dc94b5806bc2bc322a5/8e3ec8b6-b599-4f33-91b9-c7f4c8fcbe17)

**Comment by annieszot** [comment]
I'm doing QA 
Mobile:
- [ ] iPhone: Safari 12.0
  - [ ] English
  - [ ] Spanish 
- [ ] Android: Chrome 71.0
  - [ ] English
  - [ ] Spanish 

Desktop:
- [ ] Windows 10: Chrome 71.0
  - [ ] English
  - [ ] Spanish
- [ ] Windows 7: Chrome 71.0;
  - [ ] English
  - [ ] Spanish 
- [ ] Windows 7: Internet Explorer 11
  - [ ] English
  - [ ] Spanish 
- [ ] MacOS 10.14: Chrome 71.0
  - [ ] English
  - [ ] Spanish 
- [ ] Windows 10: Edge 17.17; 
  - [ ] English
  - [ ] Spanish 
- [ ] Windows 10: Internet Explorer 11
  - [ ] English
  - [ ] Spanish

**Comment by Niicck** [comment]
alt-text is present. Not checking every browser, but I feel confident that it'll be there. Closing
